packages/react-core/src/components/Tabs/Tab.tsx

```diff
@@ -19,5 +19,23 @@ export interface TabProps extends Omit<React.HTMLProps<HTMLAnchorElement | HTMLB
   isHidden?: boolean;
 }
 
-export const Tab: React.FunctionComponent<TabProps> = () => null;
+/** The parent <Tabs> component accecesses this component's propeties directly in order to present each Tab */
+export const Tab: React.FunctionComponent<TabProps> = ({
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  children,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  className,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  href,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  title,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  eventKey,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  tabContentId,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  tabContentRef,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  isHidden
```

Comment:
Instead of disabling each line you can do:
```jsx
/* eslint-disable @typescript-eslint/no-unused-vars */
...
/* eslint-enable @typescript-eslint/no-unused-vars */
```

Reply:
> This PR should allow `react-docgen` to get `Tab`'s props. You can also just do `propComponents: [TabProps]` in the frontmatter to get them to show.

Unfortunately this PR did not fix the issue, therefore i did not remove the draft from it. I quickly checked it again now, and seems the return null is the issue. Returning anything else like an empty React.Fragment makes the props work.

Reply:
propComponents: [TabProps] also does not work, it does not parse the types any more with this.